### PR TITLE
ADEN-10686 rotator interval change

### PR DESCRIPTION
**Pull request by KonradLinkowski**

### CODE REVIEW

File: src/ad-products/utils/fmr-rotator.ts [hunk 1]
```diff
@@ -43,9 +43,10 @@ export class FmrRotator {
 
 					if (!universalAdPackage.isFanTakeoverLoaded()) {
 						slot.once(AdSlot.SLOT_VIEWED_EVENT, () => {
+							const customDelays = context.get('options.rotatorDelay');
 							setTimeout(() => {
 								this.hideSlot();
-							}, this.refreshInfo.refreshDelay);
+							}, customDelays[slot.lineItemId] || this.refreshInfo.refreshDelay);
```
Comment (dmnsobczak) [inline src/ad-products/utils/fmr-rotator.ts:49]: If somebody disable the icVariable completely then `customDelays` will be undefined, and `undefined[slot.lineItemId]` will result in error. What about:
`const customDelays = context.get('options.rotatorDelay') || {};`
or
`this.instantConfig.get('icRotatorDelay', {})`
?

File: platforms/shared/setup/base-context.setup.ts [hunk 2]
```diff
@@ -70,6 +70,7 @@ export class BaseContextSetup implements DiProcess {
 			this.instantConfig.get('icFeaturedVideoPostroll'),
 		);
 
+		context.set('options.rotatorDelay', this.instantConfig.get('icRotatorDelay'));
```
Comment (dmnsobczak) [inline platforms/shared/setup/base-context.setup.ts:73]: This is setup file for new MediaWiki+AdEngine integration (aka UCP), but we are still using old AdEngine integration - without platforms in `app` repository. We need second pull request with the same change here:
https://github.com/Wikia/app/blob/dev/extensions/wikia/AdEngine3/src/setup.js#L121

**Review by dmnsobczak — CHANGES_REQUESTED** [review]
Possible error and second PR for app repository

File: platforms/shared/setup/base-context.setup.ts (hunk 2 above)
Comment (KonradLinkowski) [reply platforms/shared/setup/base-context.setup.ts]: https://github.com/Wikia/app/pull/18246

**Review by dmnsobczak — APPROVED** [review]

**Pull request merged**